src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -245,6 +251,12 @@ private Binder GetSpeculativeBinderForAttribute(int position, AttributeSyntax at
                 return null;
             }
 
+            // PROTOTYPE(semi-auto-props): Traverse containing symbols chain, if necessary, and add tests.
+            if (binder.ContainingMemberOrLambda is SourcePropertyAccessorSymbol accessor)
```

Comment:
We'll need to traverse here as well.

Reply:
Yep there is a prototype comment. I think using `binder.ContainingMember()` will be sufficient. I'll do that in the next PR.

Reply:
I ended up adding it to this PR since other review comments were around the same part and it's a very small fix.